src/main/webapp/aboutus.html

```diff
@@ -121,9 +121,29 @@ <h2>Shanlan Li</h2>
       </div>
     </section>
 
-    <br>
-    <h2>Our Favorite Food courts!</h2>
-      <div id="map"></div>
+    <!-- Header, image and text, no background -->
+    <section class="page-section">
+      <div class="container">
+        <div class="product-item">
+          <div class="product-item-title d-flex">
+            <!-- Change ml/mr to align left/right -->
+            <div class="bg-faded p-5 d-flex ml-auto rounded">
+              <h2 class="section-heading mb-0">
+                <span class="section-heading-upper">Our Favourite Foodcourts</span>
+                <span class="section-heading-lower">Handpicked Eats!</span>
+              </h2>
+            </div>
+          </div>
+          <div id="map" class="product-item-img mx-auto d-flex rounded img-fluid mb-3 mb-lg-0"></div>
```

Comment:
`<div id="map" class="product-item-img mx-auto d-flex rounded img-fluid mb-3 mb-lg-0 w-75"></div>` 
I tried to expand by adding `w-75` for width 75% which worked, but somehow it doesn't work (whole map disappears) when trying with either height only or both width and height. Not so sure how to tinker with the buttons though

Reply:
Oh great thanks thats cool! Perhaps it's something to do with how height and width are defined to be 500px in map.css and it messes with the bootstrap styling? I'll have to test this more.

Reply:
I guess the map size is decided in the map.css now. For moving the button, I just check the API. Do you mean the control positioning? https://developers.google.com/maps/documentation/javascript/controls#ControlPositioning

Reply:
I saw something like this on bootstrap website, not sure whether this helps:

> Box-sizing
For more straightforward sizing in CSS, we switch the global box-sizing value from content-box to border-box. This ensures padding does not affect the final computed width of an element, but it can cause problems with some third party software like Google Maps and Google Custom Search Engine.

On the rare occasion you need to override it, use something like the following:

Copy
.selector-for-some-widget {
  box-sizing: content-box;
}
With the above snippet, nested elements—including generated content via ::before and ::after—will all inherit the specified box-sizing for that .selector-for-some-widget.

link: https://css-tricks.com/box-sizing/

Reply:
Thanks for all your help, I've slightly changed the size of the map. It looks like this now:

![image](https://user-images.githubusercontent.com/21040892/60113669-dfff6c00-97a4-11e9-8e74-dc1c3df1bd55.png)